demo/image_demo.py

```diff
@@ -50,8 +69,16 @@ def main():
         cfg_options=cfg_options)
 
     # init visualizer
+    model.cfg.visualizer.radius = args.radius
+    model.cfg.visualizer.alpha = args.alpha
+    model.cfg.visualizer.line_width = args.thickness
+    dataset_meta = model.dataset_meta
+    if dataset_meta.get(
+            'dataset_name') == 'coco' and args.skeleton_style == 'openpose':
+        dataset_meta = parse_pose_metainfo(
+            dict(from_file='configs/_base_/datasets/coco_openpose.py'))
```

Comment:
Should we move this piece of code into the visualizer?

Reply:
ok，Let me try